### PR TITLE
Update list_org action 

### PR DESCRIPTION
**Pull request by akihikokuroda**
The list_org takes token and uses it when the user is not specified.

### CODE REVIEW

File: actions/list_orgs.yaml [hunk 1]
```diff
@@ -9,11 +9,15 @@ parameters:
...
-    required: true
+    required: false
   base_url:
     type: "string"
     description: "GitHub API url, include /api/v3 for GitHub Enterprise."
     required: false
+  token:
+    type: "string"
+    required: false
+    description: "API token for a member of the organization"
```
Comment (makandre) [inline actions/list_orgs.yaml]: Could you add comment that this is used when `user` is not specified?

File: actions/lib/base.py [hunk 2]
```diff
@@ -62,6 +62,9 @@ def _reset(self, org):
         else:
             self._client = Github(self.token, base_url=self.base_url)
 
+    def _temp_client(self, token, base_url):
+        return Github(token, token, base_url=base_url)
```
Comment (makandre) [inline actions/lib/base.py:68]: `base_url` can be blank or `None` so we need to default it in those cases.

**Review by makandre — CHANGES_REQUESTED** [review]

File: actions/list_orgs.yaml (hunk 1 above)
Comment (akihikokuroda) [reply actions/list_orgs.yaml]: OK.  I add it.

File: actions/lib/base.py (hunk 2 above)
Comment (akihikokuroda) [reply actions/lib/base.py:68]: OK.  I add the code.

Comment (makandre) [reply actions/lib/base.py:68]: Need to also check for `''` (blank) also and I think we should default to `DEFAULT_API_URL` if the config does not have the value

Comment (akihikokuroda) [reply actions/lib/base.py:68]: OK. I add it.

**Review by makandre — APPROVED** [review]
Thanks.

**Pull request merged**